src/librustdoc/html/markdown.rs

```diff
@@ -797,11 +797,46 @@ impl LangString {
         Self::parse(string, allow_error_code_check, enable_per_target_ignores, None)
     }
 
+    fn split(string: &str) -> impl Iterator<Item = &str> {
+        string
+            .split(|c| c == ',' || c == ' ' || c == '\t')
+            .map(str::trim)
+            .filter(|token| !token.is_empty())
+    }
+
     fn parse(
         string: &str,
         allow_error_code_check: ErrorCodes,
         enable_per_target_ignores: bool,
         extra: Option<&ExtraInfo<'_, '_>>,
+    ) -> LangString {
+        let new = Self::parse_inner(
+            string,
+            allow_error_code_check,
+            enable_per_target_ignores,
+            extra,
+            false,
+        );
+
+        let old = Self::parse_inner(
+            string,
+            allow_error_code_check,
+            enable_per_target_ignores,
+            extra,
+            true,
+        );
+
+        assert_eq!(new, old, "New token split differs from old token split");
+
+        new
+    }
+
+    fn parse_inner(
```

Comment:
I think most of this can been reverted now that we've done a crater run.

Reply:
Done!